simplified-tests/src/test/java/org/nypl/simplified/tests/pdf/PdfViewerProviderTest.kt

```diff
@@ -15,30 +15,30 @@ class PdfViewerProviderTest {
   fun supportsPdfBooksWhenEnabled() {
     val preferences = ViewerPreferences(
       flags = mapOf(
-        "enablePDFJSReader" to true
+        "enableOldPDFReader" to true
       )
     )
 
     val book = Mockito.mock(Book::class.java)
     val format = Mockito.mock(BookFormat.BookFormatPDF::class.java)
     val provider = PdfViewerProvider()
 
-    Assertions.assertTrue(provider.canSupport(preferences, book, format))
+    Assertions.assertFalse(provider.canSupport(preferences, book, format))
   }
 
   @Test
   fun doesNotSupportPdfBooksWhenDisabled() {
     val preferences = ViewerPreferences(
       flags = mapOf(
-        "enablePDFJSReader" to false
+        "enableOldPDFReader" to false
```

Comment:
I think we need a comment here to explain that setting `enableOldPDFReader` to false _enables_ the _current_ (pdf.js) reader. Otherwise it will be confusing.